tests/test_web_urldispatcher.py

```diff
@@ -47,14 +59,17 @@ async def test_access_root_of_static_handler(
     show_index: bool,
     status: int,
     prefix: str,
+    request_path: str,
     data: Optional[bytes],
 ) -> None:
     # Tests the operation of static file server.
     # Try to access the root of static file server, and make
     # sure that correct HTTP statuses are returned depending if we directory
     # index should be shown or not.
-    my_file = tmp_path / "my_file"
-    my_dir = tmp_path / "my_dir"
+    # Ensure that html in file names is escaped.
+    # Ensure that links are url quoted.
+    my_file = tmp_path / "<img src=0 onerror=alert(1)>.txt"
+    my_dir = tmp_path / "<img src=0 onerror=alert(1)>.dir"
     my_dir.mkdir()
```

Comment:
Looks like these paths aren't valid, shall we change the test to Linux only, or create a new test?